bin/razor-binary-wrapper

```diff
@@ -9,6 +9,24 @@ RAZOR_HOME='/opt/puppetlabs/server/apps/razor-server/share/razor-server'
 # code you are running -- so fails for anything, say, in the path. :/
 export BUNDLE_GEMFILE="${RAZOR_HOME}/Gemfile"
 
+# Load the sysconfig file for the service, primarily
+# to get the RAZOR_CONFIG setting.
+if [ -e /etc/sysconfig/pe-razor-server ]; then
+  sysconfig='/etc/sysconfig/pe-razor-server'
+elif [ -e /etc/sysconfig/razor-server ]; then
+  sysconfig='/etc/sysconfig/razor-server'
+fi
```

Comment:
This is simple enough I think I'd side with it living here, instead of pushing more stuff into the vanagon project. Especially if that's just sedding stuff.

Reply:
@mwaggett Would this be a more vanagon-driven approach?
- Have vanagon create /etc/sysconfig/razor-admin
- Load contents of that file to source in /etc/sysconfig/razor-server or /etc/sysconfig/pe-razor-server depending on build type

Reply:
If I'm understanding you correctly, that seems.. kind of roundabout? But I'm not really sure what razor-admin is used for and where everything needs to get sourced. 

It's still a little unclear to me why certain things live in razor-vanagon, e.g. https://github.com/puppetlabs/razor-vanagon/tree/master/resources/files, and some live in razor-server, e.g. https://github.com/puppetlabs/razor-server/tree/master/ext. 

Another option is erb templates for allthethings, but this all might be more trouble than it's worth for a project that is not being officially prioritized. ¯\_(ツ)_/¯

Reply:
`razor-admin` is a binary that runs outside of Razor (user runs it at will), but should have the same system configuration as either razor-server or pe-razor-server, depending on if it's FOSS or PE.

I'm happy to have those razor-server/ext files live in razor-vanagon, doesn't make a difference to me. `razor-admin` is a ruby executable though, so it seems to make sense that it'd live in the razor-server project. I'm just not clear on options to bridge razor-server's file to what razor-vanagon configures.

Reply:
We do a bunch of sedding to manipulate razor-server files for pe-razor-server: https://github.com/puppetlabs/razor-vanagon/blob/master/configs/components/razor-server.rb#L75-L95 (which is probably not the best way to do all this, but that's what we've got for now).

And I agree that razor-admin should probably continue to live in the razor-server project.

but idk, I'm tempted to say don't worry about it until this project gets appropriately prioritized..  ¯\_(ツ)_/¯